app.py

```diff
@@ -6,3 +6,15 @@
 @app.route('/')
 def home():
     return render_template("home.html")
+
+@app.route('/help')
```

Comment:
I'd rename this route and the associated template to clarify that this is for 'help choosing'. A future developer might look at this and think it's a generic help page for the application.